### PR TITLE
Test runtime attach

### PR DESCRIPTION
**Pull request by felixbarny**
Does not work for JBoss, Jetty and Tomcat because they are missing `jps` or the `tools.jar`.

### CODE REVIEW

File: integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/AbstractServletContainerIntegrationTest.java [hunk 1]
```diff
@@ -155,6 +167,10 @@ protected boolean isDeployViaFileSystemBind() {
         return true;
     }
 
+    protected boolean runtimeAttach() {
```
Comment (eyalkoren) [inline integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/AbstractServletContainerIntegrationTest.java]: Is this used?

File: integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/JettyIT.java [hunk 2]
```diff
@@ -42,11 +42,6 @@
 
     public JettyIT(final String version) {
         super(new GenericContainer<>("jetty:" + version)
-                .withEnv("JAVA_OPTIONS", "-javaagent:/elastic-apm-agent.jar")
```
Comment (eyalkoren) [inline integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/JettyIT.java]: Could it be that Jetty is not tested at all?

**Comment by eyalkoren** [comment]
Not sure I follow everything here. Let's discuss.

**Comment by felixbarny** [comment]
Closing in favor of #775

**Pull request closed without merging**